src/AspNet.Security.OAuth.Vkontakte/VkontakteAuthenticationHandler.cs

```diff
@@ -27,6 +28,11 @@ public class VkontakteAuthenticationHandler : OAuthHandler<VkontakteAuthenticati
             [NotNull] ISystemClock clock)
             : base(options, logger, encoder, clock)
         {
+            if (string.IsNullOrEmpty(Options.ApiVersion))
```

Comment:
Consider removing that.

Reply:
Consider moving that to a OAuthOptions.Validate() override. And do not remove this check entirely since the parameter is required by the VK API.

Reply:
That's why I suggested doing `["v"] = Options.ApiVersion ?? VkontakteAuthenticationDefaults.ApiVersion` to safely get rid of the check :smile:

Reply:
This one doesn't guard against an empty string.

Reply:
`["v"] = !string.IsNullOrEmpty(Options.ApiVersion) ? Options.ApiVersion : VkontakteAuthenticationDefaults.ApiVersion` then.